Synview-backend/middleware/auth_middleware.ts

```diff
@@ -1,15 +1,12 @@
-import {create, verify} from "@zaubrik/djwt";
+import { create, verify } from "@zaubrik/djwt";
```

Comment:
this still is not right - this file should not import `djwt` - all usage of `djwt` should live in `JWTHelpers.ts`